website/content/docs/secrets/aws.mdx

```diff
@@ -118,6 +118,36 @@ management tool.
     EOF
     ```
 
+    Finally, the `policy_document` argument supports the same
+    [identity templating](https://www.vaultproject.io/docs/concepts/policies#templated-policies)
+    as for Vault policies. This enables advanced use cases such as restricting resources
+    based on claims provided via JWT authentication. For example with Gitlab:
+
+    ```text
+    $ vault write aws/roles/my-other-role \
+        policy_arns=arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess,arn:aws:iam::aws:policy/IAMReadOnlyAccess \
+        iam_groups=group1,group2 \
+        credential_type=iam_user \
+        policy_document_template=true \
```

Comment:
I thought on my first attempt that `policy_document_template` will be the actual policy, maybe if this is a a bool something like `enable_templating` or something would be better to understand?

Reply:
I went with `enable_policy_document_templating`, it's a big long-ish but at least it's clear!